tox/config.py

```diff
@@ -1059,8 +1059,20 @@ def __init__(self, reader, crossonly=False):
         self.reader = reader
         self.crossonly = crossonly
 
-    def do_replace(self, x):
-        return self.RE_ITEM_REF.sub(self._replace_match, x)
+    def do_replace(self, value):
+        '''
```

Comment:
I think that the PEP style guide requires docstrings to be using double quotes - this here is just a multi-line string at the start of the function